apps/examples/testcase/le_tc/network/itc_net_netdb.c

```diff
@@ -0,0 +1,69 @@
+/****************************************************************************
+
+*
+* Copyright 2017 Samsung Electronics All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+* either express or implied. See the License for the specific
+* language governing permissions and limitations under the License.
+*
+****************************************************************************/
+
+/// @file itc_net_netdb.c
+/// @brief ITC Test Case Example for getaddrinfo() API
+
+#include <sys/types.h>
+#include <sys/socket.h>
+#ifdef CONFIG_LIBC_NETDB
```

Comment:
How about to move checking a netdb features from a source file to a Kconfig file.

Reply:
I think it's correctas I refer utc also.